src/components/pager/pager.component.js

```diff
@@ -33,17 +33,17 @@ const Pager = (props) => {
   function sizeSelector() {
     return (
       <Dropdown
-        options={ props.pageSizeSelectionOptions }
-        value={ props.pageSize }
-        onChange={ (ev) => { props.onPagination('1', ev.target.value, 'size'); } }
+        options={props.pageSizeSelectionOptions}
```

Comment:
The build is failing as our linter requires spaces after `{` and before `}` for props,  please can you update these

Reply:
My IDE settings have been outdated, but now fixed it :)